### DISCUSSION TITLE
Use Rectangle shortcuts to _select_ next window

### DISCUSSION THREAD

**Discussion by jschulenklopper** [rxhanson/Rectangle#331] [Ideas]
I love Rectangle, and the ease of resizing and positioning the active window to screen areas. I wished that the Rectangle keyboard shortcut pattern (`Ctrl-Option-` plus key) was available to _select_ or activate other windows.

Currently, we need to select another window using `Cmd-Tab` (or `Cmd-backtick` for another window of the same application). How convenient would it be that keyboard shortcuts like `Ctrl-Option-` plus `[` or `]` would be available to select / activate another window without using the mouse or trackpad, and without using the `Cmd-Tab` window cycler?

Ideally, I'd like that an app selects the window that I'm currently _looking at_ but I understand that that is too advanced yet :-)

**Comment by rxhanson** [comment]
I did look into this, but ran into an issue with deciding which window to activate next. This sounds like a trivial problem, but it is actually pretty complicated when you factor in multiple windows with varying levels of overlap. It could just ignore windows that are underneath any windows, but that's a subpar/confusing experience when there's an expectation of just switching windows.

**Reply by jschulenklopper** [reply]
Is the list of applications, or application windows available from the OS, ordered by LRU? Perhaps that'd be sufficient: just cycle to the next window in the queue, and show that, thus ignoring whether windows are visible, overlapping or hidden (in the dock).

What I dislike about the default macOS window or application switcher is that a) hidden windows cannot be made visible/active easily, and b) it ignores multiple windows of the same application. Perhaps Rectangle could step into that niche - just a thought (but without knowing if this is feasible).

**Comment by thomasfr** [comment]
if i understood you correctly, i think thats possible out of the box in macos
![image](https://user-images.githubusercontent.com/287432/206750246-df0d8f65-34aa-4bd7-b4a7-637e9e946282.png)
